1.preprocess_data/1.preprocess_training_movies.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+jupyter nbconvert --to python preprocess_training_data.ipynb
+python preprocess_training_data.py
```

Comment:
```suggestion
python preprocess_training_data.py

```

Files need an empty line at the end